jipstatus.py

```diff
@@ -31,12 +31,15 @@ def add_domain(user):
 
 def enumerate_updates(jira):
     user = cfg.args.user
+    project = cfg.args.project
 
     since = datetime.datetime.now() - datetime.timedelta(days=int(cfg.args.days))
 
-    jql = "(project = QLT OR assignee in membersOf('linaro-landing-team-qualcomm')) AND updatedDate > -%sd" % cfg.args.days
+    jql = "updatedDate > -%sd" % cfg.args.days
```

Comment:
aren't we missing the 'OR assignee in ...' with this change? 
The idea of the query was to get all cards assigned to a specific project and all cards assigned to team members.

Reply:
A few lines below there is a line for filtering using 'assignee = $USER'. However, that would only be used if there is a username as provided  with the './jipstatus.py --user $USER' flag. Which may be a problem.

What are your thoughts?

Reply:
it's a different purpose, since it's a OR vs a AND. 
If you want to filter only your updates, you will use "-u" and you get only your updates.
If you want to see all the changes which relates to the landing team, you need project = QLT to get updates for all *our* cards, and you need the OR is member of landing team, to get updates for cards assigned to us, but which are not in the QLT project (I think I needed that to get your CAM updates ;))

to be honest, the reason why we didn't try to upstream this tool is because we didn't find the right way to express the JQL in a generic way :) 

I am fine with making any change in the args or params, but we really need to support (at least) the 2 use cases above.

Reply:
Just saw you comment after posting the below update. I think I need to meditate upon this further.

Reply:
What about adding a `--teams linaro-landing-team-qualcomm` flag. We'd still have the AND/OR issue, but we can solve that with some admittedly cludgy if/elif statements.

```
if teams && project
    jql += "project = QLT or assignee in membersOf('linaro-landing-team-qualcomm')"
elif project
    jql += "project = QLT"
    jql += ' AND assignee = "%s"' % add_domain(user)
else
    jql += ' AND assignee = "%s"' % add_domain(user)
```

Reply:
that seems reasonable to have project and team args. 
I think the ( ) in the JQL are important, it's ( PJT or TEAM ) and UPDATES and USER
If we add project and team arguments, then we should support either --project, or --team (only), or --project xx --team yy

Reply:
If you're happy with the below, I'll code it up and create a function wrapping the basic jql query creation.

```
if team && project
    jql += "(project = $PROJECT or assignee in membersOf('$TEAM'))"
elif team
    jql += "assignee in membersOf('$TEAM')"
elif project
    jql += "project =  $PROJECT"
    jql += ' AND assignee = "%s"' % add_domain($USER)
else
    jql += ' AND assignee = "%s"' % add_domain($USER)
```